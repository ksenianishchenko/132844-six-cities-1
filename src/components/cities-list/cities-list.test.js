import React from "react";
import renderer from "react-test-renderer";
import CitiesList from "./cities-list.jsx";

it(`Cities list renders correctly`, () => {
  const cities = [{
    city: `Moscow`,
  }];

  const tree = renderer
  .create(<CitiesList
    cities = {cities}
    onclick = {() => {}}
  />);

  expect(tree).toMatchSnapshot();
});
