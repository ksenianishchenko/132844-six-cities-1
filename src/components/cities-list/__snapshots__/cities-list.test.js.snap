// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cities list renders correctly 1`] = `
<section
  className="locations container"
>
  <ul
    className="locations__list tabs__list"
  >
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item"
        href="#"
        onClick={[Function]}
      >
        <span>
          Moscow
        </span>
      </a>
    </li>
  </ul>
</section>
`;
